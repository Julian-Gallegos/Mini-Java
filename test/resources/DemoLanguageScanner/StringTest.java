a = " 'Four score and seven years ago' ";
b = " 'our fathers brought forth upon this continent, a new nation' ";
System.out.println(a+b);