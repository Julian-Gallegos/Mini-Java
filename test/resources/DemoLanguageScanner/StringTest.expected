ID(a) BECOMES TEXT(" 'Four score and seven years ago' ") SEMICOLON ID(b) BECOMES TEXT(" 'our fathers brought forth upon this continent, a new nation' ") SEMICOLON STDOUT LPAREN ID(a) PLUS ID(b) RPAREN SEMICOLON